examples/SConscript

```diff
@@ -44,7 +45,7 @@ prgs = []
 
 for e, extralibs in examples:
     prgs.append(env.Program(e, LIBS=[
-        'boost_thread',
+        'boost_thread-mt',
```

Comment:
I have no clue;) But http://stackoverflow.com/questions/3031768/boost-thread-linking-boost-thread-vs-boost-thread-mt seems to suggest it's a symlink anyway .. let's try it.